kuryr_kubernetes/controller/managers/prometheus_exporter.py

```diff
@@ -117,8 +120,30 @@ def _record_ports_quota_per_subnet_metric(self):
             ports_availability = total_num_addresses-ports_count
             self.port_quota_per_subnet.labels(**labels).set(ports_availability)
 
+    def _record_lbs_metrics(self):
+        """Records the number of members available per LB and the LB state"""
+        critical_lbs = [
+                ('dns-default', 'openshift-dns'),
+                ('kubernetes', 'default')]
+        for name, namespace in critical_lbs:
+            klb = utils.get_kuryrloadbalancer(name, namespace)
+            lb = klb.get('status', {}).get('loadbalancer', {})
+            lb_id = lb.get('id')
+            if not lb_id:
```

Comment:
I wonder if missing KLB is a problem?

Reply:
this wouldn't cause any problems:
`{}.get('status', {}).get('loadbalancer', {}).get('id')`
And eventually the controller takes care of creating a CRD, in case that was deleted manually by the user.

Reply:
Yeah, I noticed we'll just ignore it. Okay, makes sense, it's controller's job to take care of this.